spec/findbugs_translate_checkstyle_format/translate_spec.rb

```diff
@@ -34,4 +102,44 @@
       end
     end
   end
+
+  describe 'set_dummy' do
+    context 'single SrcDir' do
+      xml = {
+        'BugCollection' => {
+          'Project' => {
+            'SrcDir' => 'test_dir'
+          }
+        }
+      }
+      require 'rexml/document'
+      doc = REXML::Document.new
+      checkstyle = doc.add_element("checkstyle")
+      before { set_dummy(xml, checkstyle) }
+      it 'return blank dom' do
+        expect(doc.get_elements('/checkstyle/file/error')).to be_empty
+        expect(doc.get_elements('/checkstyle/file').first.attribute('name').value).to eq('test_dir')
+      end
+    end
+    context 'many SrcDir' do
+      xml = {
+        'BugCollection' => {
+          'Project' => {
+            'SrcDir' => [
+              'test_dir1',
+              'test_dir2'
+            ]
+          }
+        }
+      }
+      require 'rexml/document'
+      doc = REXML::Document.new
+      checkstyle = doc.add_element("checkstyle")
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.